src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/SecretStorage/VaultStorage.php

```diff
@@ -0,0 +1,224 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\FunctionalTestingFramework\DataGenerator\Handlers\SecretStorage;
+
+use Magento\FunctionalTestingFramework\Config\MftfApplicationConfig;
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+use Magento\FunctionalTestingFramework\Util\Logger\LoggingUtil;
+use Vault\Client;
+use VaultTransports\Guzzle6Transport;
+
+class VaultStorage extends BaseStorage
+{
+    const MFTF_PATH = '/mftf';
+    /**
+     * Adobe Vault
+     */
+    const BASE_PATH = '/dx_magento_qe';
```

Comment:
So on local dev, the base path is actually `/secret`. Should this be a configurable field in .env? Otherwise, I cannot use a local vault server.

Tagging @okolesnyk as well for this, this is more of a feature determination.

Reply:
Changed this to be configurable in .env